crux-http-server/src/crux/http_server.clj

```diff
@@ -34,6 +36,33 @@
 ;; ---------------------------------------------------
 ;; Utils
 
+(defn- raw-html
+  [{:keys [body title options]}]
+  (str (hiccup2/html
+        [:html
+         {:lang "en"}
+         (into
+          [:head
```

Comment:
nesting looks a bit weird here - looks like `[:body]` is within `[:head]`, and I can't see what the `into`'s doing?